_posts/2021/2021-04-26-order-by-created-and-updated-with-a-concern.md

```diff
@@ -0,0 +1,112 @@
+---
+title: "Order by created_at and updated_at with a concern"
+description: "If you sort models by timestamps regularly, this is nice"
+layout: article
+category: ruby
+image:
+  base: "2021/order-by-created-and-updated-with-a-concern"
+  alt: "Food Truck Festival ’ drive-by’"
+  source: "https://unsplash.com/photos/1SjD5ZEiUsA"
+  credit: "Micheile Henderson"
+---
+
+Rails provides `scope`s as a way to encapsulate your regularly used queries. I've suggested before that it's a good idea to [keep calls to the query API within models](/ruby/only-use-named-scopes-ouside-models/).
```

Comment:
* Should 'scope' be in backticks?
* *regularly-used

This paragraph feels a little of balance...maybe it needs a connector...or switching it around "I've previously suggested that ...and Rails provides...".

Reply:
```suggestion
I've suggested before that it's a good idea to [keep calls to the query API within models](/ruby/only-use-named-scopes-ouside-models/) and Rails provides `scope`s as a way to encapsulate regularly-used queries.
```